### PR TITLE
23754 Deleted obsolete statement

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#23754

*Description of changes:*
- app version = 5.12.11
- deleted "upload files will be renamed" statement per Jacqueline
- updated unit test
- fixed some whitespace

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-753-pczlfk3a.web.app

File: src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue:1]: Before:
image

After:
image

File: src/components/ContinuationIn/AuthorizationProof.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/AuthorizationProof.vue:1]: Before:
image

After:
image

File: src/components/ContinuationIn/AuthorizationProof.vue [hunk 1]
```diff
@@ -76,7 +76,6 @@
         v-for="(document, index) in authorization.files"
         :key="document.fileKey"
         class="upload-file-row mt-5"
-        :class="{ 'mb-n2': (index < numFiles) }"
```
Comment (severinbeauvais) [inline src/components/ContinuationIn/AuthorizationProof.vue:79]: Before:
image

After:
image

File: src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue [hunk 2]
```diff
@@ -85,10 +85,6 @@
               </v-icon>
             </v-btn>
           </div>
-
-          <p class="info-text font-14 mt-6 mb-0">
-            Uploaded files will be renamed automatically.
-          </p>
```
Comment (severinbeauvais) [inline src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue:91]: Before:
image

After:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-753-pczlfk3a.web.app

**Review by ArwenQin — APPROVED** [review]
Looks good!

**Pull request merged**